app/views/admin/users/gdpr_delete_requests/index.html.erb

```diff
@@ -1,41 +1,80 @@
 <% if FeatureFlag.enabled?(:member_index_view) %>
-  <div class="crayons-card p-3 s:p-4 m:p-7">
-    <header class="flex flex-col l:flex-row justify-content-between l:items-center">
+  <div class="crayons-card overflow-admin-main-layout-padding p-4 m:p-0">
+    <header class="flex flex-col l:flex-row justify-content-between l:items-center p-0 m:p-7 pb-4">
       <h1 class="crayons-title ml-3 l:ml-0">Members<span class="screen-reader-only"> (GDPR Delete Requests)</span></h1>
       <%= render "admin/users/index/tabs" %>
     </header>
-    <div class="crayons-notice mb-4" aria-live="polite">
-      <strong>These users' accounts were deleted. Please, destroy GDPR data from external services (GA, Mailchimp) for them.</strong>
+    <div class="flex justify-end px-2 m:px-7">
+      <%= paginate @gdpr_delete_requests, theme: "admin", scope: @gdpr_delete_requests, label: "Paginate GDPR delete requests", context: "top" %>
     </div>
-    <table class="crayons-table" width="100%">
-      <thead>
+    <div id="table-description" class="my-4  max-w-75 color-base-60 px-2 m:px-7">
+     These accounts have been deleted from the community. Ensure that any data held in external databases or mailing lists is deleted before marking as "Deleted."
```

Comment:
**Quick suggestion and clarification question:** Should this read "Done" rather than "Deleted" here to match the ["Done" checkmarks](https://github.com/forem/forem/pull/17287/files#diff-552115e2d2a819ee1cf273837722d65f5a57b02768413ca9f758e1809f2143daR34) next to each user (as shown below)? Asking because the designs show this as reading "Deleted" here in addition to it reading "Deleted" next to the checkmarks. I'm fine with either term, so long as the two terms match. 😊 

<img width="957" alt="Screen Shot 2022-04-14 at 12 42 05 PM" src="https://user-images.githubusercontent.com/32834804/163455927-2f2c207f-d9cf-4928-b7d0-0a67523fb40a.png">


```suggestion
     These accounts have been deleted from the community. Ensure that any data held in external databases or mailing lists is deleted before marking as "Done."
```

Reply:
Oh good spot! I'll change the actions to "Deleted" as per the designs 😄